test/unit/org/apache/cassandra/io/sstable/metadata/MetadataSerializerTest.java

```diff
@@ -122,53 +124,46 @@ public File serialize(Map<MetadataType, MetadataComponent> metadata, MetadataSer
 
         TableMetadata cfm = SchemaLoader.standardCFMD("ks1", "cf1").build();
         MetadataCollector collector = new MetadataCollector(cfm.comparator)
-                                          .commitLogIntervals(new IntervalSet<>(cllb, club));
+                                      .commitLogIntervals(new IntervalSet<>(cllb, club));
 
         String partitioner = RandomPartitioner.class.getCanonicalName();
         double bfFpChance = 0.1;
         return collector.finalizeMetadata(partitioner, bfFpChance, 0, null, false, SerializationHeader.make(cfm, Collections.emptyList()));
     }
 
-    @Test
-    public void testMaReadMa() throws IOException
-    {
-        testOldReadsNew("ma", "ma");
-    }
-
-    @Test
-    public void testMaReadMb() throws IOException
-    {
-        testOldReadsNew("ma", "mb");
-    }
-
-    @Test
-    public void testMaReadMc() throws IOException
+    private void testVersions(String... versions) throws Throwable
```

Comment:
I propose `testOldReadsNew` name (and similarly for `testOldReadsNewMVersions` and `testOldReadsNewNVersions`). This is the crux of the test, but with the current name it's hidden in the very details of the implementation.

Reply:
ok